gems/pending/MiqVm/MiqVm.rb

```diff
@@ -32,18 +32,23 @@ def initialize(vmCfg, ost=nil)
         # If we're passed a snapshot ID, then obtain the configration of the
         # VM when the snapshot was taken.
         #
+        # TODO: move to MiqVmwareVm
         if (@vim = @ost.miqVim)
             $log.debug "MiqVm::initialize: accessing VM through server: #{@vim.server}" if $log.debug?
             @vimVm = @vim.getVimVm(vmCfg)
             $log.debug "MiqVm::initialize: setting @ost.miqVimVm = #{@vimVm.class}" if $log.debug?
             @ost.miqVimVm = @vimVm
             @vmConfig = VmConfig.new(@vimVm.getCfg(@ost.snapId))
+        # TODO: move this to MiqRhevmVm.
         elsif (@rhevm = @ost.miqRhevm)
             $log.debug "MiqVm::initialize: accessing VM through RHEVM server" if $log.debug?
+            $log.debug "MiqVm::initialize: vmCfg = #{vmCfg}"
             @rhevmVm = @rhevm.get_vm(vmCfg)
             $log.debug "MiqVm::initialize: setting @ost.miqRhevmVm = #{@rhevmVm.class}" if $log.debug?
             @ost.miqRhevmVm = @rhevmVm
-            @vmConfig = VmConfig.new(@rhevmVm.getCfg(@ost.snapId))
+            @vmConfig = VmConfig.new(getCfg(@ost.snapId))
+            $log.debug "MiqVm::initialize: @vmConfig.getHash = #{@vmConfig.getHash.inspect}"
+            $log.debug "MiqVm::initialize: @vmConfig.getDiskFileHash = #{@vmConfig.getDiskFileHash.inspect}"
```

Comment:
I guess this was code copied from the `MiqVolumeManager` at some point, but the log messages weren't updated. The lines should be changed, but I wouldn't change all the log lines at this point. Given `_log` is specific to the app, I don't think it should be used in the code under gems/pending. We need to define a general way to integrate vmdb logging with gems that are not vmdb-specific. At least, we should do that before they're spun off into proper gems.